### ISSUE TITLE
Release/tag 3.0.1 is missed in `galaxy.yaml` in the branch `stable-3`

### ISSUE THREAD

**Issue by yurnov** [ansible-collections/kubernetes.core#709]
##### SUMMARY
Release/tag 3.0.1 is missed in `galaxy.yaml` in the branch `stable-3`

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME
Ansible Galaxy

##### ANSIBLE VERSION
Not related to ansible/ansible-core version, tested with ansible-core version 2.16.6:
```
ansible --version
ansible [core 2.16.6]
  config file = None
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.12/site-packages/ansible
  ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/local/bin/ansible
  python version = 3.12.2 (main, Feb  7 2024, 21:40:00) [GCC 12.2.0] (/usr/local/bin/python)
  jinja version = 3.1.4
  libyaml = True
```

##### COLLECTION VERSION
```
# /root/.ansible/collections/ansible_collections
Collection      Version
--------------- -------
kubernetes.core 3.0.0
root@ec13eacdd5bd:/#
```
in the same time:
```
root@ec13eacdd5bd:/# cat  /root/.ansible/collections/ansible_collections/kubernetes/core/CHANGELOG.rst | head -n 10
===================================
Kubernetes Collection Release Notes
===================================

.. contents:: Topics

v3.0.1
======

Release Summary
root@ec13eacdd5bd:/#
```

##### CONFIGURATION
not related 

##### OS / ENVIRONMENT

not related

##### STEPS TO REPRODUCE
Install `kubernetes.core` collections from `stable-3` branch:
```
root@ec13eacdd5bd:/# cat collactions.yaml
collections:
- name: https://github.com/ansible-collections/kubernetes.core.git
  type: git
  version: stable-3
root@ec13eacdd5bd:/#
root@ec13eacdd5bd:/# ansible-galaxy collection install -r collactions.yaml
Starting galaxy collection install process
Process install dependency map
Cloning into '/root/.ansible/tmp/ansible-local-6291icd6htc/tmp1z77x0m1/kubernetes.core2xr08s6o'...
remote: Enumerating objects: 8536, done.
remote: Counting objects: 100% (907/907), done.
remote: Compressing objects: 100% (432/432), done.
remote: Total 8536 (delta 502), reused 764 (delta 414), pack-reused 7629
Receiving objects: 100% (8536/8536), 4.51 MiB | 8.51 MiB/s, done.
Resolving deltas: 100% (5465/5465), done.
branch 'stable-3' set up to track 'origin/stable-3'.
Switched to a new branch 'stable-3'
Starting collection install process
Installing 'kubernetes.core:3.0.0' to '/root/.ansible/collections/ansible_collections/kubernetes/core'
Created collection for kubernetes.core:3.0.0 at /root/.ansible/collections/ansible_collections/kubernetes/core
kubernetes.core:3.0.0 was installed successfully
root@ec13eacdd5bd:/#
```
Check that the output will show version 3.0.0 instead of the expected 3.0.1.

Check output of:
```
 ansible-galaxy collection list kubernetes.core

# /root/.ansible/collections/ansible_collections
Collection      Version
--------------- -------
kubernetes.core 3.0.0
```

Alternative way: check the content of [galaxy.yaml](https://github.com/ansible-collections/kubernetes.core/blob/stable-3/galaxy.yml) in `stable-3` branch

##### EXPECTED RESULTS
output of `ansible-galaxy` will shown the version 3.0.1 (latest mentioned in `CHANGELOG.txt` in `stable-3` branch) when installing collection from `stable-3` branch

Output of  `ansible-galaxy collection list kubernetes.core` should shown `kubernetes.core 3.0.1`

##### ACTUAL RESULTS
output of `ansible-galaxy`  shown the version 3.0.0, even in `CHANGELOG.txt` mentioned 3.0.1 when collection is installed from in `stable-3` branch

Output of  `ansible-galaxy collection list kubernetes.core` should shown `kubernetes.core 3.0.0`

**Comment by yurnov** [comment]
I can prepare PR with a fix for this issue, it's an easy one

**Comment by yurnov** [comment]
Fixed in release [3.1.0](https://github.com/ansible-collections/kubernetes.core/releases/tag/3.1.0), closing ticket
